app/assets/javascripts/angular_apps/common/promise_factory.js

```diff
@@ -2,7 +2,7 @@
   'use strict';
 
   angular
-    .module('ungc.dashboard')
+    .module('ungc.admin')
```

Comment:
why does common module only belong to admin module. What is the significance of common out here.